api/common.go

```diff
@@ -15,11 +15,11 @@ import (
 	"github.com/docker/libtrust"
 )
 
+// Version of Current REST API
+var Version = version.Version("1.22")
```

Comment:
Either way, we shouldn't encourage the practice of setting a global in an unsafe manner that affects the behavior of the code base. Not sure why this can't be a field on the client object or something else.